Play article / Play summary for individual URL

dylano opened the pull request:
@tamarahills Not ready for landing as I will probably add #43 into this PR as well, but wanted to get your input on the approach so far -- particularly on usage of async/await. Does that muddy the waters when the rest of the code is using vanilla promises?

tamarahills commented on command/texttools.js:
```diff
@@ -22,7 +22,7 @@ var texttools = {
   cleanText: function(htmlStr) {
     // Remove the HTML marks.
-    var strippedHtml = htmlStr.replace(/<[^>]+>/g, '');
+    var strippedHtml = htmlStr.replace(/<[^>]+>/g, ' ');
```
Are you sure about this change?  Was this a code formatting from the prettifier or an actual change to fix a bug.  Thanks.

tamarahills commented on command/polly_tts.js:
```diff
@@ -11,7 +11,7 @@ var polly_tts = {
         OutputFormat: 'mp3',
         SampleRate: '16000',
-        VoiceId: 'Kimberly'
+        VoiceId: process.env.POLLY_VOICE || 'Kimberly'
```
Thanks for doing this.

dylano commented:
OK, this one is ready to go now.

_Notes:_
* Added POST routes `/article` and `/summary` to CommandController. These take an article URL in the body and return the URL of the resulting audio file
* refactorings:
  * replaced all var with const/let
  * created helper functions for common text/audio code blocks
